### PR TITLE
initialized dotenv config to work w/ sample.env and added nodemon as …

### PR DESCRIPTION
**Pull request by garzeah**
### What this PR does (required):
- Initialized dotenv config to work with our sample.env file
- Installed nodemon as a dev. dependency

### Screenshots / Videos (required):
- Not applicable

### Any information needed to test this feature (required):
- Not applicable

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const connectDB = require("./db");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+require("dotenv").config({ path: './sample.env' });
```
Comment (moffatethan) [inline server/app.js]: The `sample.env` is important to other collaborators to have. This is just what the `.env` should have. By ignoring the sample.env onboarding future collaborators would be more complicated. I would revert this and just create a `.env` on your environment.

**Review by moffatethan — COMMENTED** [review]
Left you some comments to address, thank you!

File: server/app.js (hunk 1 above)
Comment (garzeah) [reply server/app.js]: Ah, okay thanks for the feedback Ethan! I just implemented your changes. Also I just realized that we have nodemon already, should I revert that as well?

**Review by moffatethan — APPROVED** [review]
LGTM!

**Pull request merged**